build.gradle

```diff
@@ -10,6 +10,7 @@ repositories {
 }
 
```

Comment:
Can you please specify the build java version information? (sourceCompatibility, targetCompatibility)

Reply:
fixed